app/scripts/modules/core/src/presentation/main.less

```diff
@@ -524,6 +524,9 @@ dl {
 
 .tooltip-inner {
   overflow-wrap: break-word;
+  p:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
only kind of related to this PR; `<p>` tags have a 10px bottom margin, which makes them look bad in tooltips, especially when it's the only thing in the tooltip.

Reply:
omg thank you I think I noticed this a while ago and got irritated but moved on with my life...